models/time_cache.cr

```diff
@@ -0,0 +1,26 @@
+struct TimeCache(K, V)
```

Comment:
I was thinking of adding a pagination crawler (not a great name), that goes and gets all the repos. What do you think?